Azure/connectors/azure_blob.py

```diff
@@ -95,6 +96,7 @@ async def get_azure_blob_data(self) -> list[str]:
         blob_list = self.azure_blob_wrapper().list_blobs()
         _last_modified_date = self.last_event_date
         records: list[Any] = []
+        limit_of_events_to_push = int(os.getenv("AZURE_BATCH_SIZE", 10000))
```

Comment:
Cannot we move the definition of `limit_of_events_to_push` in `__init__`? In order to avoid having to read the env variable each time.

Reply:
I thought about this
In current workflow we can change env varibale on the fly. Maybe it will be more useful for debug purposes